pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -873,6 +886,90 @@ func registriesConfigIgnition(templateDir string, controllerConfig *mcfgv1.Contr
 	return &registriesIgn, nil
 }
 
+func (ctrl *Controller) createAddInheritableCapabilitiesMC() error {
+	var configMapName = "crio-add-inheritable-capabilities"
+
+	// Check if the crio-add-inheritable-capabilities config map exists in the openshift-machine-config-operator namespace
+	addInheritableCapabilitiesCM, err := ctrl.kubeClient.CoreV1().ConfigMaps(ctrl.namespace).Get(context.TODO(), configMapName, metav1.GetOptions{})
+	capabilitiesCMIsNotFound := errors.IsNotFound(err)
+	if err != nil && !capabilitiesCMIsNotFound {
+		return fmt.Errorf("error checking for %s config map: %v", configMapName, err)
+	}
+	// If the crio-add-inheritable-capabilities config map exists, that means the crio-add-inheritable-capabilities MC was already created
+	// so we should not create this MC again and return
+	if addInheritableCapabilitiesCM != nil && !capabilitiesCMIsNotFound {
+		return nil
+	}
+
+	sel, err := metav1.LabelSelectorAsSelector(metav1.AddLabelToSelector(&metav1.LabelSelector{}, builtInLabelKey, ""))
+	if err != nil {
+		return err
+	}
+	// Find all the MachineConfigPools
+	mcpPoolsAll, err := ctrl.mcpLister.List(sel)
+	if err != nil {
+		return err
+	}
+
+	// Create the crio-add-inheritable-capabilities MC for all the available pools
+	for _, pool := range mcpPoolsAll {
```

Comment:
I think this should be enough for master/worker pools, since all custom pools inherit from worker. Put another way, I'm not sure we want to tack generated MCs into custom pools created by the user.

Reply:
would `ctrl.getPoolsForContainerRuntimeConfig` get that?

Reply:
I think that gets all pools as well, although looking at https://github.com/openshift/machine-config-operator/commit/0c8e86938e06aea2d686ae950bb31b0f9b5bfc2d, it seemed that we didn't do that, so, hmm, I guess it's not a big issue either way?

I seem to recall we did have a similar config thing with only master/worker pools but I could be mis-remembering. This isn't really blocking so it's more so up to you :) I just thought that it was unnecessary to generate for custom pools and could mess up user configuration

Reply:
yeah I agree it would be better just with master and worker but I'm fine leaving it as-is